app/extensions/brave/locales/en-US/extensions.properties

```diff
@@ -29,3 +29,5 @@ torrent= Torrent Viewer
 torrentDesc=Uses WebTorrent to display torrents directly in the browser. Supports torrent files and magnet links.
 vimium= Vimium
 vimiumDesc=
+honey=Honey
```

Comment:
The in-progress fix for this is here:

https://github.com/brave/muon/compare/chrome-cookies-getall

This should either suppress that error outright or should be able to be modified in some way to do so. I think it is waiting on further testing.

I'm on Slack if anyone with an interest in this wants to discuss.